### PR TITLE
Update Passcode Layout

### PR DESCRIPTION
**Pull request by theck13**
### Fix
Update the passcode layout with the new design.  Note that the number pad in the tablet layout is slightly lower in the ***After*** screenshot compared to the ***Design*** mockup.  The number pad is not only centered in the layout, but also assigned below the "Enter your passcode" text to ensure there is no overlap between those views.  As a consequence, the number pad is slightly lower.  Also, the error message location was changed from the top of the screen to the default location of `Toast` components for consistency.  All references to "PIN" were changed to "passcode" in the interface.  The `colorPrimary` attribute was added to the `Theme.Simplestyle.Passcode` style so that the app bar color would be blue rather than purple, which is the default for `Theme.MaterialComponents.Light.NoActionBar`, in the recent apps view.  See the screenshots below for illustration.

update_passcode_layout_again_phone

update_passcode_layout_again_tablet

### Test
The fingerprint icon shown below the 7 is only visible if the device has fingerprint authentication hardware.  If the device does not have that hardware, the icon is not shown.
1. Open navigation drawer.
2. Tap ***Settings*** item in navigation drawer.
3. Notice ***Passcode Lock*** section uses "passcode" rather than "PIN" references.
4. Tap ***Turn passcode lock on*** item under ***Passcode Lock*** section.
5. Enter passcode.
6. Enter passcode again.
7. Notice app layout is as screenshots show above.

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release
`RELEASE-NOTES.txt` was updated in 4ddf8a46 with:
> Updated passcode lock design

### CODE REVIEW

**Comment by SylvesterWilmott** [comment]
Looks great!

When choosing a new passcode can we use the string `Choose a 4 digit passcode` instead of `Enter your passcode` otherwise we don't have any hint about the 4 digit limit.

File: Simplenote/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -96,20 +96,6 @@
...
-    <string name="passcode_enter_old_passcode">Enter your old PIN</string>
-    <string name="passcode_re_enter_passcode">Re-enter your PIN</string>
-    <string name="passcode_change_passcode">Change PIN</string>
-    <string name="passcode_set">PIN set</string>
-    <string name="passcode_wrong_passcode">Wrong PIN, please try again.</string>
-    <string name="passcode_preference_title">PIN Lock</string>
-    <string name="passcode_turn_off">Turn PIN lock off</string>
-    <string name="passcode_turn_on">Turn PIN lock on</string>
-    <string name="passcode_summary">Screenshots are disabled while PIN lock is on</string>
-    <string name="description_fingerprint_supported">Fingerprint Supported</string>
```
[the remark this answers is not in the document]
Comment (theck13) [reply Simplenote/src/main/res/values/strings.xml]: Resources removed from the `values/string.xml` should be removed from all translated `values-*/strings.xml` files automatically during the release process.  @loremattei, can you confirm that?

**Comment by theck13** [comment]
Unfortunately, there is only one string used for both creating your passcode when one doesn't exist yet and verifying your passcode when the app is locked.  Therefore, the "Enter your passcode" string was chosen since it works better in both cases.

File: Simplenote/src/main/res/values/strings.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (loremattei) [reply Simplenote/src/main/res/values/strings.xml]: Yes, this is right. They will be removed from GlotPress when the new `values/string.xml` is uploaded during the code freeze and, so, they will be removed from all translated `values-*/strings.xml` when the new translations are download from GlotPress during the final release.

**Comment by SylvesterWilmott** [comment]
>Unfortunately, there is only one string used for both creating your passcode when one doesn't exist yet and verifying your passcode when the app is locked. Therefore, the "Enter your passcode" string was chosen since it works better in both cases.

No worries! Everything else looks great

File: Simplenote/src/main/res/values/strings.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (anitaa1990) [reply Simplenote/src/main/res/values/strings.xml]: Thank you both for the clarification! TIL! In the woo app, I've been removing all strings from the translated files every time 🤦‍♀! This is good to know :)

**Pull request merged**